iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/topology/ReplaceTeePointByVoltageLevelOnLine.java

```diff
@@ -137,84 +137,82 @@ public void apply(Network network, NamingStrategy namingStrategy, boolean throwE
 
         // tee point is the voltage level in common with tpLine1, tpLine2 and tpLineToRemove
         VoltageLevel teePoint = TopologyModificationUtils.findTeePoint(tpLine1, tpLine2, tpLineToRemove);
-        if (teePoint == null) {
-            noTeePointAndOrTappedVoltageLevelReport(reportNode, oldLine1Id, oldLine2Id, lineToRemoveId);
-            LOGGER.error("Unable to find the tee point and the tapped voltage level from lines {}, {} and {}", oldLine1Id, oldLine2Id, lineToRemoveId);
-            if (throwException) {
-                throw new PowsyblException(String.format("Unable to find the tee point and the tapped voltage level from lines %s, %s and %s", oldLine1Id, oldLine2Id, lineToRemoveId));
-            } else {
-                return;
-            }
-        }
 
-        // tapped voltage level is the voltage level of tpLineToRemove, at the opposite side of the tee point
-        VoltageLevel tappedVoltageLevel = tpLineToRemove.getTerminal1().getVoltageLevel() == teePoint
-                ? tpLineToRemove.getTerminal2().getVoltageLevel()
-                : tpLineToRemove.getTerminal1().getVoltageLevel();
+        if (teePoint != null) {
```

Comment:
In this method, it would be more readable with the following test at the beginning:

```java
if (teePoint == null) {
    noTeePointAndOrTappedVoltageLevelReport(reportNode, oldLine1Id, oldLine2Id, lineToRemoveId);
    ModificationLogs.logOrThrow(throwException, String.format("Unable to find the tee point and the tapped voltage level from lines %s, %s and %s", oldLine1Id, oldLine2Id, lineToRemoveId));
    return;
}
```
(Which corresponds to restore the original order of the nullability test of `teePoint`.)